css-view-transitions-1/Overview.bs

```diff
@@ -1210,7 +1210,7 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 				1. Set |height| to the current height of |capturedElement|'s [=new element=]'s [=border box=].
 
-				1. Set |transform| to a transform that maps the |capturedElement|'s [=new element=]'s [=border box=] from document origin to its quad from the [=snapshot viewport origin=].
```

Comment:
I'm a bit confused by what was meant here so maybe I'm just misunderstanding but "its quad from the snapshot viewport origin" is what we're trying to compute here so it seems a bit circular.

I also think the "border box from document origin" is wrong if we were to load the page scrolled (since the snapshot viewport is effectively fixed but the document origin is pushed up by the scroll)...instead we want to go from the snapshot rect origin to "element's border box relative to the snapshot rect origin".